inst/startup/merger.R

```diff
@@ -33,108 +38,116 @@ for (package in azbatchenv$packages) {
 }
 
 parent.env(azbatchenv$exportenv) <- globalenv()
-sessionInfo()
 
 enableCloudCombine <- azbatchenv$enableCloudCombine
 cloudCombine <- azbatchenv$cloudCombine
 
 if (typeof(cloudCombine) == "list" && enableCloudCombine) {
-  results <- vector("list", batchTasksCount)
-  count <- 1
+  if (!require("doParallel", character.only = TRUE)) {
+    install.packages(c("doParallel"), repos = "http://cran.us.r-project.org")
+    require("doParallel", character.only = TRUE)
+    library("doParallel")
+  }
+
+  sessionInfo()
+  cluster <- parallel::makeCluster(parallel::detectCores(), outfile = "doParallel.txt")
+  parallel::clusterExport(cluster, "libPaths")
+  parallel::clusterEvalQ(cluster, .libPaths(libPaths))
+
+  doParallel::registerDoParallel(cluster)
 
   status <- tryCatch({
-    if (errorHandling == "remove" || errorHandling == "stop") {
+    count <- 1
+
+    # Verify if merge task is sub or final task
```

Comment:
Similar to before - should we be pulling this out to another change for multi-merge?

Reply:
Yes removed